tests/test_lithium.py

```diff
@@ -697,6 +697,30 @@ def interesting(sub, conditionArgs, tempPrefix):  # pylint: disable=no-self-argu
                 else:
                     self.assertEqual(f.read(), valid_reductions[-1])
 
+    def test_minimize_collapse_braces(self):
+        class Interesting(DummyInteresting):
+
```

Comment:
nit: extra space

Reply:
Not sure which space you're referring to here.  As far as I can tell, this is the same format as the other tests.